A little changes in README for Copilot language server

rauf322 opened the pull request:
I believe there was some confusions in README how to install Copilot-language-server, correct me if i'm wrong

rauf322 committed:
Fix Copilot Language Server setup instructions

- Updated to use mason-lspconfig.nvim instead of mason.nvim
- Corrected package name from 'copilot-language-server' to 'copilot'
- Removed unnecessary nvim-lspconfig manual setup
- Added clear installation steps and important notes
- Clarified that blink-copilot handles LSP integration automatically

rauf322 committed:
Update Copilot sign-in command and package name

Updated authentication command and clarified package name.

fang2hou commented:
`mason-lspconfig.nvim` is not a must plugin in my mind (at least I am not using it)

maybe `mason-lspconfig` or `mason + custom lspconfig below`

```
{
  "neovim/nvim-lspconfig",
  opts = {
    servers = {
      copilot = {},
    },
  }
```

rauf322 committed:
Update ReadMe

rauf322 committed:
Revise README for Copilot installation steps

Updated installation instructions and setup options for Copilot Language Server.

rauf322 commented:
Yes you are correct i forgot that mason-lspconfig not necessary to be, i added 2 options to installation with mason-lspconfig and without it and using custom lspconfig

rauf322 committed:
fix typo

fang2hou commented:
Thanks!
